src/main/java/com/github/tomakehurst/wiremock/client/BasicMappingBuilder.java

```diff
@@ -94,6 +94,12 @@ public BasicMappingBuilder withHeader(String key, StringValuePattern headerPatte
     return this;
   }
 
+  @Override
+  public ScenarioMappingBuilder withHeader(String key, MultiValuePattern headerPattern) {
```

Comment:
We should probably be consistent and return `BasicMappingBuilder` here.

Reply:
done